mw-webapp/src/dataAccessLogic/UserPreviewDAL.ts

```diff
@@ -10,11 +10,18 @@ import {PartialWithUuid} from "src/utils/PartialWithUuid";
  */
 export class UserPreviewDAL {
 
+  /**
+   * Get amount of all users in collection
+   */
+  public static async getUsersPreviewAmount(): Promise<number> {
+    return await UserService.getUsersDTOAmount();
+  }
+
   /**
    * Get all UserPreview
    */
-  public static async getUsersPreview(): Promise<UserPreview[]> {
-    const usersDTO = await UserService.getUsersDTO();
+  public static async getUsersPreview(uuid?: string): Promise<UserPreview[]> {
```

Comment:
you canot get user preview with no uuid

Reply:
I don't understand comment. We need to pass uuid for fetch data partly (pagination)

Reply:
don't sort users by uuid

Reply:
investigate what will happened when fields will be the same

Reply:
We need to get snapshot for possibility to use pagination. For getting snapshot we need uuid of document